terra_notebook_utils/cli/__init__.py

```diff
@@ -23,6 +25,15 @@ def load(cls):
     def write(cls):
         with open(cls.path, "w") as fh:
             fh.write(json.dumps(cls.info, indent=2))
+
+    @classmethod
+    def resolve(cls, default_workspace: str, default_namespace: str):
+        workspace = default_workspace or (cls.info['workspace'] or WORKSPACE_NAME)
+        namespace = default_namespace or (cls.info['workspace_google_project'] or WORKSPACE_GOOGLE_PROJECT)
+        if workspace and namespace is None:
+            from terra_notebook_utils.workspace import get_workspace_namespace
```

Comment:
Why is this imported inside of the method?  Just a nit I suppose.

Reply:
Loading modules lazily may improve the responsiveness of the CLI.